app/routes/companyInterest/components/CompanyInterestList.tsx

```diff
@@ -55,10 +71,25 @@ const RenderCompanyActions = ({
     {clickedCompanyInterest === id ? 'Er du sikker?' : 'Slett'}
   </Button>
 );
+export const EVENT_TYPE_OPTIONS = [
+  { value: '', label: 'Vis alle arrangementstyper' },
+  { value: 'company_presentation', label: 'Bedriftspresentasjon' },
+  { value: 'course', label: 'Kurs' },
+  { value: 'breakfast_talk', label: 'Frokostforedrag' },
+  { value: 'lunch_presentation', label: 'Lunsjpresentasjon' },
+  { value: 'bedex', label: 'BedEx' },
+  { value: 'digital_presentation', label: 'Digital presentasjon' },
+  { value: 'other', label: 'Alternativt arrangement' },
+  { value: 'sponsor', label: 'Sponser' },
+  { value: 'start_up', label: 'Start-up kveld' },
+  { value: 'company_to_company', label: 'Bedrift-til-bedrift' },
+];
```

Comment:
Is there a reason you have redefined this and not imported it from another place to reduce duplicates? f. ex. from here https://github.com/webkom/lego-webapp/blob/0b7c6c58bf622c7922c24d2745f5ad7df54c6a86/app/routes/companyInterest/components/CompanyInterestPage.tsx

Reply:
On it.